src/main/java/com/alyrow/gdx/particle/ParticleSystem.java

```diff
@@ -65,41 +68,46 @@
 
     /**
      * Manage physics on particles
+     *
      * @see PhysicManager
      */
     private PhysicManager physicManager;
 
     /**
      * Manage modifier on particles
+     *
      * @see ModifierManager
      */
     private ModifierManager modifierManager;
 
     /**
      * Constructor that create a particle system
-     * @param type Particles type (Halo, Texture, No texture)
-     *             {@link ParticleType}
-     * @param world If `null` collisions between particles and world's objects are deactivated
+     *
+     * @param type   Particles type (Halo, Texture, No texture)
+     *               {@link ParticleType}
+     * @param world  If `null` collisions between particles and world's objects are deactivated
      * @param camera For display particles on the screen
      */
-    public ParticleSystem(int type, World world, Camera camera) {
+    public ParticleSystem(ParticleType type, World world, Camera camera) {
         this.type = type;
         this.world = world;
         this.camera = camera;
         batch = new SpriteBatch();
         time = TimeUtils.millis();
 
-        if(type == ParticleType.HALO) texture = new ParticleTexture("halo.png");
-        else if (type == ParticleType.NOTHING) texture = new ParticleTexture();
+//        if(type == ParticleType.HALO) texture = new ParticleTexture("halo.png");
+//        else if (type == ParticleType.NOTHING) texture = new ParticleTexture();
+        texture = this.type.getTexture();
```

Comment:
Need to do the breaking case

Reply:
False alert
https://github.com/alyrow/box2d-light-particle/commit/a6a176eb1898c9fef3f38168ed68ed7699223ff7